core/src/main/java/org/apache/iceberg/rest/responses/ErrorResponse.java

```diff
@@ -22,18 +22,17 @@
 import java.io.StringWriter;
 import java.util.Arrays;
 import java.util.List;
-import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.rest.RESTResponse;
 
 /** Standard response body for all API errors */
 public class ErrorResponse implements RESTResponse {
 
   private String message;
   private String type;
-  private int code;
+  private Integer code;
```

Comment:
I think this needs to still be `int`, since `code` is required? https://github.com/apache/iceberg/blob/master/open-api/rest-catalog-open-api.yaml#L1108 , which makes sense we should always have some known status code.

Reply:
Hmm, we only require `code` to be there, we also don't check for `message` and `type`